static/src/js/components/ProjectPanels/AccessMethod.js

```diff
@@ -14,102 +14,155 @@ import './AccessMethod.scss'
-              value="Download"
-              checked
+export const AccessMethod = ({
+  index,
+  metadata,
+  onSelectAccessMethod,
+  onSetActivePanel
+}) => {
+  const {
+    id,
+    tags
+  } = metadata
+
+  const handleAccessMethodSelection = (method) => {
+    console.log('handleAccessMethodSelection', method)
+    onSelectAccessMethod({
+      collectionId: id,
+      selectedAccessMethod: method
+    })
+  }
+
+  const collectionCapibilities = tags['edsc.extra.serverless.collection_capabilities']
+
+  const { data = {} } = collectionCapibilities
+  const { granule_online_access_flag: downloadable } = data
+
+  const radioList = []
+
+  if (downloadable) {
+    radioList.push((
+      <Radio
+        id={`${id}_access-method__direct-download`}
+        name={`${id}_access-method__direct-download`}
+        key={`${id}_access-method__direct-download`}
+        value="Download"
+        onClick={() => handleAccessMethodSelection('download')}
+      >
+        Direct Download
+        <OverlayTrigger
+          placement="right"
+          overlay={(
+            <Tooltip
+              className="tooltip--large tooltip--ta-left"
             >
-              Direct Download
-              <OverlayTrigger
-                placement="right"
-                overlay={(
-                  <Tooltip
-                    className="tooltip--large tooltip--ta-left"
-                  >
-                    Direct download of all data associated with the selected granules.
-                    The desired data will be available for download immediately.
-                    Files will be accessed from a list of links displayed in the
-                    browser or by using a download script.
-                  </Tooltip>
-                )}
-              >
-                <i className="access-method__radio-tooltip fa fa-info-circle" />
-              </OverlayTrigger>
-            </Radio>
-            <Radio
-              id={`${id}_access-method__stage-for-deliver`}
-              name={`${id}_access-method__stage-for-deliver`}
-              value="Stage For Delivery"
+              Direct download of all data associated with the selected granules.
+              The desired data will be available for download immediately.
+              Files will be accessed from a list of links displayed in the
+              browser or by using a download script.
+            </Tooltip>
+          )}
+        >
+          <i className="access-method__radio-tooltip fa fa-info-circle" />
+        </OverlayTrigger>
+      </Radio>
+    ))
+  }
+
+  if (Object.keys(tags).indexOf('edsc.extra.serverless.subset_service.echo_orders') !== -1) {
+    radioList.push((
+      <Radio
+        id={`${id}_access-method__stage-for-deliver`}
+        name={`${id}_access-method__stage-for-deliver`}
+        key={`${id}_access-method__stage-for-deliver`}
+        value="Stage For Delivery"
+        onClick={() => handleAccessMethodSelection('echoOrder')}
```

Comment:
What is `echoOrder`? The casing doesn't match the tag postfix and isn't JSON friendly. The other methods seem to use a downcased JSON friendly string here.

Reply:
echoOrder is just the key that I'm storing (or going to store) the access method configs in the redux store, so it is a JS key